src/components/views/project/ProjectHeader.tsx

```diff
@@ -7,17 +8,43 @@ import styled from 'styled-components';
 const ProjectHeader = (props: { project: IProject }) => {
 	const { title, verified, image, adminUser, traceCampaignId } =
 		props.project;
+
+	const [adjustTitle, setAdjustTitle] = useState<boolean>(false);
 	const name = adminUser?.name;
 	const traceable = !!traceCampaignId;
 
+	useEffect(() => {
+		const threshold = 280;
+		const updateScrollDir = () => {
+			const scrollY = window.pageYOffset;
+
+			if (scrollY > threshold && !adjustTitle) {
+				setAdjustTitle(true);
+			}
+			if (scrollY < threshold) {
+				setAdjustTitle(false);
+			}
+		};
+
+		const onScroll = () => {
+			window.requestAnimationFrame(updateScrollDir);
+		};
+
+		window.addEventListener('scroll', onScroll);
+
+		return () => window.removeEventListener('scroll', onScroll);
+	}, [adjustTitle]);
+
```

Comment:
I think it's better to use intersectionObserver instead of scroll event to detect some thing is on view port or not